packages/DropdownMenu/src/Item.styles.js

```diff
@@ -0,0 +1,52 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const itemStyles = css`
+  ${stylers.fontSize()};
+  ${stylers.lineHeight()}
+  cursor: pointer;
+  display: block;
+  font-family: ${tokens.fontFamily.default};
+  font-weight: normal;
+  padding: ${tokens.spaceSm} ${stylers.spacer(2)};
+  text-decoration: none;
+
+  &:focus {
+    box-shadow: ${tokens.highlight.active.withBorder.boxShadow}
+    outline: none;
+    position: relative;
+    ${stylers.z(1)}
+  }
+
+  ${({ isDestructive, isDisabled }) => {
+    let colorString;
+    if (isDisabled) {
+      colorString = isDestructive ? "orangeDisabled" : "blackDisabled";
+      return { color: tokens.color[colorString] };
+    }
+
+    colorString = isDestructive ? "orangeDarken10" : "colorBlack";
+    return { color: tokens.color[colorString] };
+  }}}
+
+  &:hover{
```

Comment:
minor, but i feel like there needs to be a space after hover
`hover {` `active {`